docs/conf.py

```
@@ -0,0 +1,175 @@
+
+# Client-side rendering: the directive emits the diagram source and the browser draws it,
+# so no headless browser or diagram binary has to be present to build the HTML.
+mermaid_output_format = 'raw'
+
+# -- HTML output -------------------------------------------------------------------------
+
+# The theme the `docs` extra installs, and the one ReadTheDocs serves.
+html_theme = 'sphinx_rtd_theme'
+
+
+def _module_names_under(root):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_module_names_under`

(ANN202)